build-support/bin/mypy.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import argparse
+import subprocess
+from typing import List
+
+from common import die
+
+
+def main():
+  targets = create_parser().parse_args().targets
+  run_mypy(targets)
+
+
+def create_parser() -> argparse.ArgumentParser:
+  parser = argparse.ArgumentParser(description="Run MyPy with our config file.")
+  parser.add_argument(
+    "targets",
+    default=["build-support::", "src/python::", "tests/python::", "contrib::"],
+    nargs="*",
+    help="Pants targets, e.g. `tests::`.",
+  )
+  return parser
+
+
+def run_mypy(targets: List[str]) -> None:
+  command = [
+    "./pants",
+    "mypy",
+    "--mypy-mypy-version=0.701",
+    "--config-file=build-support/mypy/mypy.ini",
```

Comment:
These two options should probably be in `pants.ini`. And if `0.701` is valuable, we should consider bumping the default mypy version so that we don't need to override it here.

Reply:
That is a substantially simpler solution to just use `pants.ini`... I honestly didn't think to do that because I was basing this all of the `isort.sh` script.

Took your suggestion and removed the script! Thanks for the review :)